sycl/test-e2e/Regression/alloc_with_invalid_alignment.cpp

```diff
@@ -0,0 +1,77 @@
+// REQUIRES: level_zero
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+//==----- alloc_with_invalid_alignment.cpp - SYCL USM allocation test-----==//
```

Comment:
I don't think it is required to have a license boiler plate in tests, that's the case even with the upstream LLVM.